### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by RussKie**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2681)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+    A clear and concise description of any alternative solutions or features you've considered.
+  -->
+
+**Will this feature affect UI controls?**
+<!--
+    Yes / No
+
+    If yes, please describe the following:
+
+        - Will VS Designer need to support the feature? If yes, describe how you expect it to funtion.
```
Comment (vladimir-krestov) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Typo
```suggestion
        - Will VS Designer need to support the feature? If yes, describe how you expect it to funсtion.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+<!-- e.g. 3.0 Preview1, or daily build number, use `dotnet --info` -->
+
+* Have you experienced this same bug with .NET Framework?: 
+<!-- Yes / No -->
+
 **Problem description:**
 
-**Actual behavior:** <!-- callstack for crashes / exceptions -->
+<!--
+    Describe the current behaviour, what is wrong or not working as expected.
```
Comment (vladimir-krestov) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Typo
```suggestion
    Describe the current behavior, what is wrong or not working as expected.
```

Comment (RussKie) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Well it isn't, it is British and Australian English 😎

**Comment by RussKie** [comment]
> Do we want to update the contribution guidelines too?

[The guide](https://github.com/dotnet/winforms/blob/master/Documentation/contributing.md) has the following wording, which I think is sufficient:

> _Please file an issue for any larger change you would like to propose. Please choose the template most appropriate for the issue type._

I'm happy to update/alter it, if you think it is necessary. Suggestions are welcome.

**Pull request merged**